rafx-framework/src/visibility/camera_object.rs

```diff
@@ -0,0 +1,221 @@
+use crate::visibility::visibility_object_allocator::{CameraObjectId, SlotMapArc};
+use crate::RafxResult;
+use crossbeam_channel::Sender;
+use glam::Vec3;
+use parking_lot::{RwLock, RwLockReadGuard};
+use rafx_api::RafxError;
+use rafx_visibility::{
+    AsyncCommand, CameraHandle, Projection, RegionHandle, VisibilityQuery, VisibilityWorldArc,
+};
+use slotmap::SlotMap;
+use std::sync::Arc;
+
+#[derive(Clone)]
+pub struct CameraObjectHandle {
```

Comment:
I think we should consider some other names instead of camera. I would expect a camera to be a more engine-level thing that would include more tangible, artistic/PBR parameters that drive rendering (exposure, lens settings, dof, etc.). This is a fairly abstract system. In other words, I would expect end-user code (or even we create a separate high level system) to create its own thing called a Camera (likely bindable to render to a viewport on a window) and it would own this more abstract "view/query" object. (Similarly to how lights own these objects.) Maybe something with the word Frustum in it?

Reply:
Will rename to `VisibilityFrustum`